packages/ponder-schema/tsup.config.ts

```diff
@@ -0,0 +1,15 @@
+import { defineConfig } from "tsup";
+
+export default defineConfig({
+  entry: ["src/ponder.schema.ts"],
+  platform: "neutral",
+  format: ["esm"],
+  target: "es2022",
+  bundle: true,
+  splitting: false,
+  sourcemap: true,
+  // dts: true,
```

Comment:
@tk-o enabling this causes `ponder-schema` not to build with tsup. It looks like `ponder` needs to export `Relations` and `Table` from Drizzle that we can then re-export, perhaps, I'm not sure.

Reply:
I think it's an issue coming from `drizzle-orm`. We might need to skip declaration type generation for `ponder-schema` package until the underlying problem is fixed. Possibly related:
- https://github.com/drizzle-team/drizzle-orm/pull/4097

Reply:
@tk-o perhaps we can wait a few days for this to be merged in and see if it resolves our issue, or figure out a workaround that's not pretty. Third option would be to merge this as is now, and then come back to enable this option once the PR above is merged.

Reply:
Drizzle is a downstream dependency, we don't import it directly. I assume it's gonna take some weeks before Ponder team decides to update Drizzle dependency, and we get the updateded `ponder` package to use in our repo.